C# Examples/EposServiceConsumer/UnitTests/OldEposTest.cs

```diff
@@ -0,0 +1,376 @@
+﻿using NUnit.Framework;
+using RD.EposServiceConsumer.Helpers;
+using RD.EposServiceConsumer.Helpers.BusinessObjects;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+
+namespace RD.EposServiceConsumer.UnitTests
+{
+    /// <summary>
+    /// Summary description for EposTest
+    /// </summary>
+    [TestFixture]
+    public class OldEposTest
```

Comment:
Tbh - this is probably pretty much all they really need - i.e. how to get the restaurant details, create a booking, etc.